zilliqa/src/pool.rs

```diff
@@ -124,11 +124,6 @@ impl TransactionPool {
                 continue;
             };
 
-            // If we've popped a nonced transaction, that may have made a subsequent one valid
-            if let Some(next) = tx_index.next().and_then(|idx| self.transactions.get(&idx)) {
-                self.ready.push(next.into());
-            }
-
```

Comment:
@bzawisto Why did we remove this? I'm going to re-add it since I can't imagine this code working without it

Reply:
Oh I see that you rely on the push in `mark_executed` instead. Will add a clarifying comment.